Patches/OpenCV3/Patch.cmake

```diff
@@ -14,3 +14,11 @@ file(COPY ${OpenCV_patch}/graphcuts.cpp
 file(COPY ${OpenCV_patch}/OpenCVFindLibsVideo.cmake
   DESTINATION ${OpenCV_source}/cmake
 )
+
```

Comment:
Is this patch appropriate for all compilers? There is a lot of code getting patched so it's hard to tell. If we aren't sure, it might be good to use an MSVC version check around the patching itself.